azure-functions-maven-plugin/src/main/java/com/microsoft/azure/maven/function/AddMojo.java

```diff
@@ -378,6 +414,23 @@ protected void assureInputFromUser(final String prompt, final String initValue,
         }
     }
 
+    protected void assureInputInBatchMode(final String input, final Function<String, Boolean> validator,
+                                          final Consumer<String> setter, final boolean required)
+            throws MojoFailureException {
+        if (validator.apply(input)) {
+            info(FOUND_VALID_VALUE);
+            setter.accept(input);
+            return;
```

Comment:
unnecessary return? or you can remove below else to avoid nesting if. Leave the decisions to u.

Reply:
Sure, removing below else to avoid nesting if.